### DISCUSSION TITLE
SqlBuilder.ToString does not include quotation marks

### DISCUSSION THREAD

**Discussion by gerardbalaoro** [maxtoroq/DbExtensions#60] [Q&A]
```vb.net
Dim Builder As New SqlBuilder
Builder.INSERT_INTO("people(name)").VALUES(TextBox1.Text)
```
Produces:
```sql
INSERT INTO people(name) VALUES (Tony Stark)
```
I'm using the string format function for substitution, as I cannot find the documentation to extract the SQL query with the values:
```vb.net
String.Format(Query.ToString, Query.ParameterValues.ToArray)
```

**Comment by maxtoroq** [comment]
Cannot reproduce, the result should be:

```text
INSERT INTO people(name)
VALUES ({0})
-- 0: System.String [Tony Stark]
```

Use the Database class to turn the query into a command, see the [tutorial](https://maxtoroq.github.io/DbExtensions/docs/SqlBuilder.html#overview).
